orchestrator/tests/security/test_nl2sql_tenancy.py

```
@@ -39,30 +39,48 @@ def _camelot_unlocatable() -> bool:  # pragma: no cover - env-dependent
-def test_query_main_database_is_deleted():
+def test_query_main_database_stays_deleted():
+    """The unscoped raw-SQL fallback must never come back, even re-enabled."""
     from modules.tools.execution import exec_research
 
     assert not hasattr(exec_research, "query_main_database")
 
 
+def test_executors_make_no_http_self_call():
+    """PRD-160 S1 grep gate: the in-process path imports no HTTP client."""
+    code = (ORCH / "modules/tools/execution/exec_research.py").read_text()
+    body = "\n".join(l for l in code.splitlines() if not l.strip().startswith("#"))
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)